session/src/main/scala/akka/stream/alpakka/cassandra/javadsl/CassandraSession.scala

```diff
@@ -70,6 +73,18 @@ final class CassandraSession(@InternalApi private[akka] val delegate: scaladsl.C
    */
   def close(executor: Executor): CompletionStage[Done] = delegate.close(ExecutionContext.fromExecutor(executor)).toJava
 
+  /**
+   * The `ProtocolVersion` used by the driver to communicate with the server.
+   */
+  def protocolVersion: CompletionStage[ProtocolVersion] =
```

Comment:
do we need this now? can't hurt

Reply:
removed, available via underlying CqlSession if needed